src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -778,9 +778,9 @@ class CreateTableStatementSegment(BaseSegment):
         Ref("IfNotExistsGrammar", optional=True),
         Ref("TableReferenceSegment"),
         Bracketed(
-            OneOf(
+            Delimited(
                 # Columns and comment syntax:
-                Delimited(
+                AnyNumberOf(
```

Comment:
Is this compatible with before? IIUC, `Delimited` means there's a separator, but `AnyNumberOf` does not.

Reply:
The outer one on line 781 should handle it?

Reply:
Got it -- thanks! Sometimes reading PRs is harder than reading code. 😛